obspy/clients/eida/__init__.py

```diff
@@ -10,120 +13,182 @@
 :license:
     GNU Lesser General Public License, Version 3
     (https://www.gnu.org/copyleft/lesser.html)
+
+Basic Usage
+-----------
+
+The first step is always to initialize a client object.
+
+>>> from obspy.clients.eida import Client
+>>> client = Client("GFZ")
+
+A client object can be initialized either with the base URL of any EIDA routing
+service or with a shortcut name that will be mapped to such a URL. The examples
```

Comment:
On 05/19/2016 01:14 PM, Tobias Megies wrote:
> In obspy/clients/eida/__init__.py
> <https://github.com/obspy/obspy/pull/1397#discussion_r63860381>:
>
>> @@ -10,120 +13,182 @@
>>  :license:
>>      GNU Lesser General Public License, Version 3
>>      (https://www.gnu.org/copyleft/lesser.html)
>> +
>> +Basic Usage
>> +-----------
>> +
>> +The first step is always to initialize a client object.
>> +
>> +>>> from obspy.clients.eida import Client
>> +>>> client = Client("GFZ")
>> +
>> +A client object can be initialized either with the base URL of any EIDA routing
>> +service or with a shortcut name that will be mapped to such a URL. The examples
>
> This is using the short URLs defined in |fdsn/header.py|, right? Would
> be good to have a link in the docstring, like |... :mod:|a shortcut name
> that will be mapped to such a URL |...|

Not sure how you want this exactly to be typeset. Like:

:mod:`a shortcut name that will be mapped to such a URL 
<obspy.clients.fdsn>`

???

Note that not all shortcut names work, only those that run the routing 
service. Except when using the even service that is not routed.

Also, some doctest fails because request to 5 nodes (default) is made in 
parallel and the order of networks in print(inventory) depends on the 
order of responses received. We cannot rely on station descriptions not 
changing either.

The fail of test_iris_example_queries_station should disappear after all 
EIDA nodes patch their FDSNWS.